### DISCUSSION TITLE
Plugin ThreeBox in Maplibre GL JS

### DISCUSSION THREAD

**Discussion by Takieddinex** [maplibre/maplibre-gl-js#1260] [Q&A]
Hi, looking for the tool or method that will allow me to load several 3D models on my map in their position and make them rotate and move. I found the Threebox plugin that seems the most suitable for my research so I wonder if it is available for Maplibre 

URL of ThreeBox : https://github.com/jscastro76/threebox/blob/master/docs/Threebox.md#loading-a-3d-model

**Comment by John2397** [comment]
I used Threebox with Mapbox GL JS v1. When I migrated from Mapbox V1 to MapLibre V1, the 3D models still worked perfectly.

**Comment by brendan33** [comment]
The latest Threebox version doesn't work with Maplibre V2 and looks like the author is not going to add support which is a shame.

https://github.com/jscastro76/threebox/issues/375
